### ISSUE TITLE
[REQUEST] MinutesToMeasureTrend_x

### ISSUE THREAD

**Issue by Cryptoqueb** [mehtadone/PTFeeder#265]
Title kinda says it all. I'd like more time variables to mesure trend, so let's say I want to know the trend for the last 20m,1h,2h, etc. Right now I'm using "MinutesToMeasureTrend" in ways that I don't think it was meant to be used (5-7m) and it's working prettty well, but I found myself not having any info on the 60m trend for example, which sucks, especially to set the current market condition. 

thx

**Comment by mehtadone** [comment]
Adding more price trends is coming. In the interim, may I ask where are you using MinutesToMeasureTrend? There are two configurable time windows available: MinutesToMeasureTrend and MinutesForLongerTermTrend.

**Comment by Cryptoqueb** [comment]
Well I just switch these two around. What I used to do is set MinutesToMeasureTrend at 7m so I could buy (regular buys and DCAs) on the very low (most of the time) using EMASPREAD has a buy strat with
   {
        "MaxPriceTrendPercentageChange": "0.2",
		   "SellOnlyMode" : "true"
      },

Instead of using trailling to get a good buy (rarerly hit the lows) I used that. The issues was that It was hard to set the market condition with a 7m trend variable. So now I used the MinutesForLongerTermTrend for this (7m) and MinutesToMeasureTrend at 60m. My market conditions now make more sense and so are my other offsets using that trend (e.i volatility).

Still a pretty unusual way to use PTF im sure, but i'm getting good resultats so far so I'll stick with that until the next big update ^^
